bcda/api/requests.go

```diff
@@ -345,6 +335,43 @@ func ValidateRequest(r *http.Request) ([]string, *fhirmodels.OperationOutcome) {
 	return resourceTypes, nil
 }
 
+func check429(jobs []models.Job, types []string, w http.ResponseWriter) ([]string, bool) {
```

Comment:
It doesn't consider basePath when checking for the 429. I think that we would still limit the end user to a single concurrent resource request at a time though. Even though they are different FHIR response types, they will still put the same pressure on our API/BFD, so it would make sense to still limit?

Reply:
I was thinking about a scenario where they want to compare v1 payload vs v2 payload... they'd have to make the calls in sequence instead of in parallel.  I'm sure either approach is fine - just throwing this out as a known limitation.

Reply:
That's a great use case. I'll go ahead and update it to distinguish between v1/v2 calls